### DISCUSSION TITLE
Docker UI and Ollama RAM use

### DISCUSSION THREAD

**Discussion by apellaman** [open-webui/open-webui#1440] [Q&A] [answered]
Hello, I'm trying to limit the ram use of VmmemWSL process on Windows11. 
Currently I could do it adding a .wslconfig file with 

[wsl2]
memory=2GB

I'm wondering which is the minimum amount of ram I can use for a run it properly and if this limit impacts the Ollama performances, or webui ones.
Thanks

**Comment by justinh-rahb** [comment, marked as answer]
If Ollama is running in Docker as well, that would be bad. If WebUI only, 2GB may work, but if it crashes you should add more accordingly.
